docs/user_guide.md

```diff
@@ -70,14 +70,26 @@ You can create a project after you signed in. Check on the "Access Level" checkb
 
 ![create project](img/new_create_project.png)  
 
-After the project is created, you can browse repositories, users and logs using the navigation tab.  
+After the project is created, you can browse repositories, members, logs, replication and configuration using the navigation tab.  
 
 ![browse project](img/new_browse_project.png)  
 
 All logs can be listed by clicking "Logs". You can apply a filter by username, or operations and dates under "Advanced Search".  

```

Comment:
not seeing advanced search on UI, please double check

Reply:
Uploaded a new image for "Advanced Search".